src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
+        let r = self + Rational::new(self.sign().into(), 2.into());
```

Comment:
whoops: `(n/d) ± (1/2) = (2n ± d)/2d`. Not sure if that's necessarily better though.